Movie App/Movie App/AppDelegate/AppDelegate.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  AppDelegate.swift
+//  Movie App
+//
+//  Created by datNguyem on 19/10/2021.
+//
+
+import UIKit
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        return true
+    }
+}
+
```

Comment:
chỉ để 1 dòng trống cuối file thôi chú
```suggestion
}
```